### ISSUE TITLE
Tiny model training?

### ISSUE THREAD

**Issue by jhkwag970** [valeoai/Halton-MaskGIT#32]
Hello, Thank you sharing your great work!

I notice that in the repo you shared checkpoint for tiny and small model which have ~20M and 60M parameters. Are these model also trained same config as paper provided?

<img width="542" height="300" alt="Image" src="https://github.com/user-attachments/assets/a5c4c405-76cb-4177-8732-e13f21c96ae0" />

Thank you!

**Comment by llvictorll** [comment]
Hello, Thanks a lot!

There are a few differences with the last pretrained networks I released: I'm running 1.5 million training steps, and for AdamW, I'm now using betas=(0.9, 0.999) with a weight_decay of 0.03, as I’ve found this setup provides better stability during training. I also disable the horizontal flipping as it does not improve the FID.

The rest remain exactly the same as in the paper (bsize, lr, warmup, etc).

Best,

Victor
